### PR TITLE
docs: source-google-analytics-data-api

### PR DESCRIPTION
**Pull request by oliviamiannone**
**Description:**

Adds docs for the new connector, source-google-analytics-data-api (GA4)

Resolves https://github.com/estuary/airbyte/issues/113

**Documentation links affected:**

The config descriptions currently contain these shortlinks:

https://go.estuary.dev/fz3SSy
https://go.estuary.dev/C9pwjr

... which are for our UA connector and should be replaced with shortlinks specific to this connector. 

Also, the documentation URL for this connector will need a new shortlink.

**Notes for reviewers:**

* Not 100% on my interpretation of custom reports, so that could use a critical eye. Everything else is tested. 
* A bot recently upgraded docusaurus, which required a node update, hence the much larger than usual changes to  package-lock.json

- - -
This change is [Reviewable](https://reviewable.io/reviews/estuary/flow/917)

### CODE REVIEW

File: site/docs/reference/Connectors/capture-connectors/google-analytics-4.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+Each is [fetched as a report](https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/runReport) and mapped to a Flow collection through a separate binding.
+
+You can also capture [custom reports](#custom-reports).
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* The Google Analytics Reporting API [enabled](https://support.google.com/googleapi/answer/6158841?hl=en) on your Google [project](https://cloud.google.com/storage/docs/projects) with which your Analytics property is associated.
```
Comment (mdibaiee) [inline site/docs/reference/Connectors/capture-connectors/google-analytics-4.md:39]: You need to enable Google Analytics Data API (as well? not sure if Reporting API is necessary)

File: site/docs/reference/Connectors/capture-connectors/google-analytics.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
-This connector captures data from a view in Google Analytics.
+This connector captures data from a view in Google Universal Analytics.
 
-:::caution
-This connector supports Universal Analytics.
-Support for Google Analytics 4 is coming soon.
+:::info
+This connector supports Universal Analytics, not Google Analytics 4.
+(We apologize for the misleading name of the Docker image below.)
```
Comment (mdibaiee) [inline site/docs/reference/Connectors/capture-connectors/google-analytics.md:7]: Actually we can rename the docker image, I'll get on that

Comment (mdibaiee) [inline site/docs/reference/Connectors/capture-connectors/google-analytics.md:7]: I'll make an attempt to rename the docker image

Comment (mdibaiee) [reply site/docs/reference/Connectors/capture-connectors/google-analytics.md:7]: Okay the connector for this one is now called `google-analytics-ua`

File: site/docs/reference/Connectors/capture-connectors/google-analytics-4.md (hunk 1 above)
Comment (oliviamiannone) [reply site/docs/reference/Connectors/capture-connectors/google-analytics-4.md:39]: Whoops, thanks for catching that

File: site/docs/reference/Connectors/capture-connectors/google-analytics.md (hunk 2 above)
Comment (oliviamiannone) [reply site/docs/reference/Connectors/capture-connectors/google-analytics.md:7]: Hmm... I don't see it renamed (https://github.com/orgs/estuary/packages) and that URL doesn't work for me

Comment (oliviamiannone) [reply site/docs/reference/Connectors/capture-connectors/google-analytics.md]: Never mind, was just missing "source" from the URL :) Would be good to have it listed on the packages page under the alias but can confirm I'm able to access the connector's page.

**Review by mdibaiee — APPROVED** [review]
Thank you Olivia!

**Pull request merged**